source/backend/api/Areas/DocumentGeneration/DocumentGenerationController.cs

```diff
@@ -66,7 +66,7 @@ public async Task<IActionResult> GetSupportedDocumentTypes()
         [ProducesResponseType(typeof(ExternalResponse<string>), 200)]
         [SwaggerOperation(Tags = new[] { "document-generation" })]
         [TypeFilter(typeof(NullJsonResultFilter))]
-        public async Task<IActionResult> UploadTemplate([FromForm] IFormFile file)
+        public async Task<IActionResult> UploadTemplate(IFormFile file)
```

Comment:
recommended by swagger - currently blocking swagger from displaying

Reply:
our swagger needs a lot of help